tools/meteor-services/deploy.js

```diff
@@ -609,6 +623,13 @@ export async function bundleAndDeploy(options) {
     });
   }
 
+  if (options.isBuildOnly) {
+    Console.info(
```

Comment:
I think it would be better to use `Console.success` here to make it clear that the process has finished.

Reply:
As we are not using success anywhere in the deploy maybe it's better to just keep as info.

Reply:
We are using error, but let's shelve this for a bigger overview then.